core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4493,7 +4321,9 @@ public static PropertyKey fromString(String input) {
    * @return all pre-defined property keys
    */
   public static Collection<? extends PropertyKey> defaultKeys() {
-    return Sets.newHashSet(DEFAULT_KEYS_MAP.values());
+    return Sets.newHashSet(DEFAULT_KEYS_MAP.values()
```

Comment:
should we instead construct `DEFAULT_KEYS_MAP` so that it doesn't contain removed keys?